code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -9,10 +9,9 @@
 
 	lock_type = /datum/locking_category/chair
 
-/obj/structure/bed/chair/New()
+/obj/structure/bed/chair/initialize()
```

Comment:
I doubt it, I compiled and tested it in game on the chairs already facing north in the CMO's office appeared above me.